contrib/apelscripts/condor_blah.sh

```diff
@@ -27,27 +27,18 @@ if [ ! -d $OUTPUT_DIR ] || [ ! -w $OUTPUT_DIR ]; then
 fi
 
 # Build the filter for the history command
-CONSTR="CompletionDate >= $yesterday && CompletionDate < $today "
+CONSTR="EnteredCurrentStatus >= $yesterday && EnteredCurrentStatus < $today && RemoteWallclockTime !=0"
```

Comment:
@JaimeFrey could you clarify the semantics of when `RemoteWallClockTime` is set to 0, when it's not, and when (if ever) it's undefined?

Reply:
I don't know the full semantics, Jaime. Suspect 0 or undefined occurs if job makes it to batch system but not to workernode. APEL is not interested in jobs that run for 0 or undefined seconds
.

Reply:
Use `=!=` to better handle undefined attrs (https://htcondor.readthedocs.io/en/latest/misc-concepts/classad-mechanism.html?highlight=classad%20mechanism#old-classad-operators)

```suggestion
CONSTR="EnteredCurrentStatus >= $yesterday && EnteredCurrentStatus < $today && RemoteWallclockTime =!=0"
```

Reply:
Now using =?=